src/core/server/legacy_compat/legacy_service.ts

```diff
@@ -157,7 +163,7 @@ export class LegacyService implements CoreService {
 
   private setupProxyListener(server: HapiServer) {
     const legacyProxy = new LegacyPlatformProxy(
-      this.logger.get('legacy', 'proxy'),
+      this.core.logger.get('legacy-proxy'),
```

Comment:
__note:__ while working on this I had to see lots of logs, and noticed that legacy logger alphabetically sorts tags that represent new logger context and it's not something we want.